ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -574,6 +574,55 @@ tests:
       EXTRA_MG_ARGS: --host-network
     workflow: openshift-e2e-aws-ovn-virt
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-metal-ipi-ovn-virt-dualstack
+  cluster: build05
+  optional: true
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      CNV_PRERELEASE_LATEST_CHANNEL: "true"
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+      EXTRA_MG_ARGS: --host-network
+      EXTRA_NETWORK_CONFIG: |
+        EXTRA_NETWORK_NAMES="extranet"
+        EXTRANET_NETWORK_SUBNET_V4='192.168.221.0/24'
+        EXTRANET_NETWORK_SUBNET_V6='fd2e:6f44:5dd8:ca56::/120'
+      TEST_SUITE: openshift/network/virtualization
+    post:
+    - chain: baremetalds-ofcir-post
+    pre:
+    - chain: baremetalds-ofcir-pre
+    - ref: baremetalds-devscripts-conf-extranetwork
+    - ref: kubevirt-install
+    workflow: baremetalds-e2e-ovn-dualstack
+  timeout: 6h0m0s
+- always_run: false
+  as: e2e-metal-ipi-ovn-virt-dualstack-techpreview
```

Comment:
oh there is one for techpreview for @kyrtapz 's EPIC? and the GA one is for @ricky-rav 's EPIC?
Requesting proper JIRA trackers and PR context update to point to specific EPICs cause it says localnet tests today in the PR describe.

Reply:
I have update the PR description that points to the jira, the idea is to have techpreview jobs too to manually run them on future features.

Reply:
title is updated, description is not really updated right? but its ok thanks